ios/utils/EpsonUtils.m

```diff
@@ -0,0 +1,72 @@
+//
+//  EpsonUtils.m
+//
+//  Created by Till POS on 14/09/21.
+//
+
+#import <Foundation/Foundation.h>
+#import "EpsonUtils.h"
+#import "PrinterSDK.h"
+
+@implementation EpsonUtils
+
++ (NSString *)makeErrorMessage:(Epos2PrinterStatusInfo *)status
+{
+    NSMutableString *errMsg = [[NSMutableString alloc] initWithString:@""];
+
+    if (status.getOnline == EPOS2_FALSE) {
```

Comment:
So epson sdk only gives code and then we add formatted message strings?

Reply:
Yes